src/ui/marker.js

```diff
@@ -33,6 +34,7 @@ type Options = {
  *   Options are `'center'`, `'top'`, `'bottom'`, `'left'`, `'right'`, `'top-left'`, `'top-right'`, `'bottom-left'`, and `'bottom-right'`.
  * @param {PointLike} [options.offset] The offset in pixels as a {@link PointLike} object to apply relative to the element's center. Negatives indicate left and up.
  * @param {string} [options.color='#3FB1CE'] The color to use for the default marker if options.element is not provided. The default is light blue.
+ * @param {number} [options.scale=1] The scale to use for the default marker if options.element is not provided.
```

Comment:
I think it might be good to specify that the default scale corresponds to a height of 41px and a width of 27px. That would give developers the information up front so they don't have to dig into the code to figure out what scale factor they want or just blindly guess.

Reply:
Great point. I'll add that to the description, and then go ahead and merge.